modules/Search/components/Subtitle.tsx

```diff
@@ -0,0 +1,31 @@
+import translations from '@prezly/themes-intl-messages';
+import { FormattedMessage } from 'react-intl';
+
+import { useAlgoliaState } from './AlgoliaStateContext';
+
+function Subtitle() {
+    const { searchState, searchResults } = useAlgoliaState();
+
+    const { query: searchQuery } = searchState;
+    const resultsCount = searchResults ? searchResults.nbHits : 0;
+
+    return (
+        <p>
+            {searchQuery ? (
+                <FormattedMessage
+                    {...translations.search.fullResultsSubTitle}
+                    values={{
+                        resultsCount: <b>{resultsCount}</b>,
+                        searchQuery: (
+                            <>
+                                &quot;<b>{searchQuery}</b>&quot;
+                            </>
+                        ),
+                    }}
+                />
+            ) : undefined}
```

Comment:
Currently, the content shifts when this text is updated.
If you want to prevent it, you need to set a height on the parent `p` element.
Otherwise, you can simplify the code by just returning `null` earlier when there is no search query

Reply:
will add a fixed height for the p element